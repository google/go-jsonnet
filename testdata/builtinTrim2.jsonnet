std.trim("string with newline character at end\n")
