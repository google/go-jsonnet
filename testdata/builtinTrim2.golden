"string with newline character at end"
